.github/workflows/flake.yml

```diff
@@ -0,0 +1,39 @@
+name: Flake
+
+# Run flake on all PRs.  This ignores many pre-existing errors however should catch
+# regressions and allows us to remove ignores and results issues iteratively.
+
+on:
+  pull_request:
+    paths:
+      - '**/*.py'
+
+jobs:
+  flake8-lint:
+    runs-on: ubuntu-latest
+    name: Lint
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Set up Python environment
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.8"
```

Comment:
Yeah probably.  I think the danger of removing this is that we might get a new python version with new checks, and as this check covers every PR it would mean everything failing.  Will change.